iot-hub/Quickstarts/back-end-application/BackEndApplication.py

```diff
@@ -40,9 +41,9 @@ def iothub_devicemethod_sample_run():
 
         input("Press Enter to continue...\n")
 
-    except IoTHubError as iothub_error:
+    except Exception as ex:
```

Comment:
@jebrando @cartertinney I couldn't figure out how to get hub specific errors (as we did previously). LMK if this needs to be more specific.

Reply:
For now the ex is the most detail we have for errors.

---
In reply to: [364444185](https://github.com/Azure-Samples/azure-iot-samples-python/pull/20#discussion_r364444185) [](ancestors = 364444185)